### PR TITLE
Ruby- Raindrops mentor notes - Removes references to prime

### PR DESCRIPTION
**Pull request by kotp**
The specification shows prime numbers as the example set of rules, but
indicates no requirement that they are special factors, only that they
are factors.

### CODE REVIEW

**Comment by mikegee** [comment]
I thought the "primes" part of the notes were out of place. This is much better! 👍

File: tracks/ruby/exercises/raindrops/mentoring.md [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+
 ```ruby
 module Raindrops
-  SOUNDS = {3 => "Pling", 5 => "Plang", 7 => "Plong"}.freeze
+  RULES =
+    {
+      3 => "Pling",
+      5 => "Plang",
+      7 => "Plong"
+    }.freeze
```
Comment (NobbZ) [inline tracks/ruby/exercises/raindrops/mentoring.md]: How is this reformatting and renaming related to primes?

File: tracks/ruby/exercises/raindrops/mentoring.md [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
 
-  def self.convert(num)
-    rhythm = SOUNDS.select{|key, sound| num % key == 0}.values
-    rhythm.empty? ? num.to_s : rhythm.join
+  def self.convert(number)
+    rhythm = RULES.select{|key, sound| number % key == 0}.values
+    rhythm.empty? ? number.to_s : rhythm.join
```
Comment (NobbZ) [inline tracks/ruby/exercises/raindrops/mentoring.md]: Same for `num` => `number`, how is this related to primes?

File: tracks/ruby/exercises/raindrops/mentoring.md [hunk 3]
```diff
@@ -1,34 +1,38 @@
...
 end
 ```
 
 ### Common suggestions
-- Using the `prime` library rather than manually calculating
+
+- Using the `prime` library rather than manually calculating doesn't make sense if we consider that prime is only suggested via examples given, but not by specification, as the specification stated is "factors".
 - Using `select`.
-- Not needing `("")` on the `join`
+- Default argument for join, if `("")` is given on the `join`.
```
Comment (NobbZ) [inline tracks/ruby/exercises/raindrops/mentoring.md]: Again, I can not see how this rephrased sentence is related to the prime removal.
